@armaniferrante compiler is complaining about:

```
error[E0599]: the method `to_account_metas` exists for struct `anchor_lang::CpiContext<'a, 'b, 'c, 'info, T>`, but its trait bounds were not satisfied
  --> programs/counter/src/lib.rs:62:1
   |
62 |   #[interface]
   |   ^^^^^^^^^^^^ method cannot be called on `anchor_lang::CpiContext<'a, 'b, 'c, 'info, T>` due to unsatisfied trait bounds
   | 
  ::: /Users/michaelhly/Projects/anchor/lang/src/context.rs:32:1
   |
32 | / pub struct CpiContext<'a, 'b, 'c, 'info, T>
33 | | where
34 | |     T: ToAccountMetas + ToAccountInfos<'info>,
35 | | {
...  |
39 | |     pub signer_seeds: &'a [&'b [&'c [u8]]],
40 | | }
   | |_- doesn't satisfy `_: anchor_lang::ToAccountMetas`
   |
   = note: the following trait bounds were not satisfied:
           `T: anchor_lang::Accounts<'_>`
           which is required by `anchor_lang::CpiContext<'_, '_, '_, 'info, T>: anchor_lang::ToAccountMetas`
   = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
   ```

but isn't that implemented here?
https://github.com/project-serum/anchor/blob/c23a5dfa29b1b4393e9e6a68d4b4c5189faf9f5c/lang/src/context.rs#L88

--- a/lang/attribute/interface/src/lib.rs
+++ b/lang/attribute/interface/src/lib.rs
@@ -197,7 +197,7 @@ pub fn interface(
             let sighash_tts: proc_macro2::TokenStream =
                 format!("{:?}", sighash_arr).parse().unwrap();
             quote! {
-                pub fn #method_name<'a,'b, 'c, 'info, T: anchor_lang::ToAccountMetas + anchor_lang::ToAccountInfos<'info>>(
+                pub fn #method_name<'a,'b, 'c, 'info, T: anchor_lang::Accounts<'info> + anchor_lang::ToAccountMetas + anchor_lang::ToAccountInfos<'info>>(
                     ctx: anchor_lang::CpiContext<'a, 'b, 'c, 'info, T>,
                     #(#args),*
                 ) -> anchor_lang::solana_program::entrypoint::ProgramResult {

